app/controllers/responsiblepeople/ConfirmAddressController.scala

```diff
@@ -69,22 +46,21 @@ class ConfirmAddressController @Inject()(override val messagesApi: MessagesApi,
                 case true
                   => Redirect(routes.CurrentAddressController.get(index))
                 case _
-                  => {
+                  =>
```

Comment:
I know it was like that before, but could you change it put for me "=>" in the same line as case followed by case action, please? Like now it just looks weird.

Reply:
lines 46-50

Reply:
Done